src/delaunay.js

```diff
@@ -174,6 +174,9 @@ export default class Delaunay {
     return buffer && buffer.value();
   }
   renderPoints(context, r = 2) {
+    if (arguments.length === 1 && ["number", "string"].includes(typeof context)) {
+      [r, context] = [+context, null];
+    }
```

Comment:
Here’s my suggestion: the more specific type is *context* (an object that implements the CanvasPathMethods API), so we only consider the first argument a context if it implements *context*.moveTo. Otherwise we assume that the first argument is *r*, and set *context* to null. But we still need to default *r* to 2 if both arguments are undefined.

```suggestion
  renderPoints(context, r) {
    if (!context || typeof context.moveTo !== "function") r = context, context = null;
    if (r === undefined) r = 2;
```

Reply:
We should probably also coerce *r* to a number (otherwise the `x + r` could result in string concatenation).

Reply:
> we only consider the first argument a context if it implements _context_.moveTo.

this breaks `renderPoints(null, 5)` which is currently only working solution for this task.

Since we're on a major version change we can include this breaking change, but if we want to be conservative we should only apply this if r === undefined